frontends/election-manager/test/render_in_app_context.tsx

```diff
@@ -81,14 +81,16 @@ export function renderRootElement(
     backend = new ElectionManagerStoreMemoryBackend(),
     logger = fakeLogger(),
     storage = new MemoryStorage(),
-    queryClient = new QueryClient(),
     apiClient = createMockApiClient(),
+    // TODO: Determine why tests fail when using createQueryClient and, by extension,
+    // QUERY_CLIENT_DEFAULT_OPTIONS
```

Comment:
One of the issues seems to be the error boundary creating a hard failure out of this otherwise silent (test-specific, as far as I can tell) error: https://github.com/votingworks/vxsuite/issues/2971. But there's definitely more going on because disabling the error boundary setting didn't address all the failures. I did verify that these issues predate the move to backend auth

Reply:
Hmm, interesting. I would definitely be wary of removing the default options completely in test, because they strongly influence the behavior of react-query and it's caching, which could potentially change the logic of the app. Maybe I'm being paranoid and you've thought it through though

Reply:
Yes totally fair! One thing worth noting is that this discrepancy between the query client in the real app vs. in tests has been around since before Auth v2 work. It was just less explicit / maybe even accidental, so I wanted to call it out now.

_index.tsx before Auth v2_

https://github.com/votingworks/vxsuite/blob/6460a1e8480fb5500a73024324c9c4b05f63e5b8/frontends/election-manager/src/index.tsx#L18-L29

_render_in_app_context.tsx before Auth v2_

https://github.com/votingworks/vxsuite/blob/43d00b66766b56e63ac39c3b9c4066b35a0efeef/frontends/election-manager/test/render_in_app_context.tsx#L81

What has changed since Auth v2 is the addition of settings, like the error boundary setting, which does increase risk. I've added this to my miscellaneous cleanup ticket so I don't lose track of it: https://github.com/votingworks/vxsuite/issues/2965

Reply:
Ah gotcha. Thanks for tracking!